src/components/TransactionList/TransactionList.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from "react";
+import { useMappedState } from "redux-react-hook";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Grid from "@material-ui/core/Grid/Grid";
+import Tooltip from "@material-ui/core/Tooltip/Tooltip";
+import TableSortLabel from "@material-ui/core/TableSortLabel/TableSortLabel";
+import DetailsModal from "./DetailsModal";
+
+import { stableSort, getSorting } from "../../helpers/helpers";
+
+const headerCols = [
+  { id: "timestamp", numeric: false, disablePadding: true, label: "Timestamp" },
+  { id: "source", numeric: false, disablePadding: false, label: "Source" },
+  {
+    id: "destination",
+    numeric: false,
+    disablePadding: false,
+    label: "Destination"
+  }
+];
+
+type Order = "asc" | "desc";
+type OrderBy = string;
+
+const mapState = (state: any) => ({
+  blokchain: state.blokchain.blocks
+});
+
+const TransactionList = (): React.ReactElement => {
+  // const dispatch = useDispatch();
+  const { blokchain } = useMappedState(mapState);
+  const [order, setOrder] = useState<Order>("asc");
+  const [orderBy, setOrderBy] = useState<OrderBy>("name");
+
+  const handleRequestSort = (event: any, property: any) => {
+    if (orderBy === property && order === "desc") {
+      setOrder("asc");
+    } else if (orderBy === property && order === "asc") {
+      setOrder("desc");
+    }
+    setOrderBy(property);
+  };
+
+  const createSortHandler = (property: any) => (event: any) => {
```

Comment:
(Wiem że ten plik to tylko zmiana nazwy ale można przy okazji poprawić parę detali)

Ta funkcja jest w zasadzie nie potrzebna, tam gdzie jest użyta można dać `onClick={e => handleRequestSort(e, row.id)}`

Reply:
Alan ja bardziej pracowałam w tym pliku i porobiłam zmiany więc Moniki wersja tego pliku nie do końca wygląda jak efekt końcowy. 
Monika myślę, że w TransactionList nic nie zmieniaj ja zrobiłam i poprawię refactor z tym co Alan sugeruje :) 
Co Panowie myślicie o tym?

Reply:
Dla mnie spoko, nie blokuje PRa - to tylko sugestie 😉widziałem te rzeczy już prędzej w kodzie ale tu akurat nadarzyła się okazja żeby łatwo to skomentować 😄

Reply:
A jeśli faktycznie na nim pracujesz to lepiej uniknąć konfliktów.